File: problems/digit_square/submissions/accepted/digit_square.java
import java.util.Scanner;

public class digit_square {
    public static void main(String[] args) {
        // INPUT
        Scanner sc = new Scanner(System.in);
        String line = sc.nextLine();
        char ch;
        int nDigit;
        String digit;
        for (int i = 0; i < line.length(); i++) {
            ch = line.charAt(i);
            digit = "" + ch;
            nDigit = Integer.parseInt(digit);
            System.out.print(nDigit * nDigit);
        }
        sc.close();
    }
}

File: problems/digit_square/submissions/accepted/digit_square.pas
program digit_square;

    var
      s1, s2, s3:string;
      i, n, code:integer;


    begin
      readln(s1);
      s2:= '';
      for i:=1 to length(s1) do
        begin
         Val(s1[i], n, code);
         n := n*n;
         Str(n, s3);
         s2 := s2+s3;
        end;
      writeln(s2);
    end.